plugins/calibre.koplugin/main.lua

```diff
@@ -59,9 +59,9 @@ function Calibre:closeWirelessConnection()
 end
 
 function Calibre:onDispatcherRegisterActions()
-    Dispatcher:registerAction("calibre_search", { category="none", event="CalibreSearch", title=_("Search in calibre metadata"), device=true,})
-    Dispatcher:registerAction("calibre_browse_tags", { category="none", event="CalibreBrowseTags", title=_("Browse all calibre tags"), device=true,})
-    Dispatcher:registerAction("calibre_browse_series", { category="none", event="CalibreBrowseSeries", title=_("Browse all calibre series"), device=true, separator=true,})
+    Dispatcher:registerAction("calibre_search", { category="none", event="CalibreSearch", title=_("Calibre search"), device=true,})
```

Comment:
Some calibre nitpickers will be upset.

Reply:
Concretely, I mean it's purposefully calibre. That's not a typo or a mistake.

Reply:
Sorry, you are absolutely right, this should be reverted.